### ISSUE TITLE
Error: Module parse failed: Unexpected character '' (1:0)

### ISSUE THREAD

**Issue by mahdidavoodi7** [jeremybarbet/next-videos#5]
Hi there,
I have a next project which I wanted to load both videos and images in it and my directory of the project is like : 

+components
|___ home.js
+statics
|___ video.mp4
|___ image.png

and my my `home.js` component is like this: 
```javascript
import React from 'react';
import styles from './styles.module.scss'

import Image1 from '../statics/image.svg'
import Video1 from '../statics/video.mp4'

export default props => {
  return (
    <div className={styles.container}>
      <img className={styles.pinguin} src={Image1}/>
      <video className={styles.video} src={Video1}/>
    </div>
  )
};
```

and my `next.config.js` is : 
```javascript
const withImages = require('next-images')
const withVideos = require('next-videos')

module.exports = {
  withImages: withImages(),
  withVideos: withVideos(),
}
```

but as you can see in the title I got this error : 

```error
./statics/video.mp4 1:0
Module parse failed: Unexpected character '' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
```

can you please help me what can I do to solve this problem?

**Comment by michaelseidel90** [comment]
Got the exact same problem :/

**Comment by mahdidavoodi7** [comment]
@michaelseidel90 have you solved this problem? if not tell me to show you how to solve it.

**Comment by michaelseidel90** [comment]
@mahdidavoodi7 You need neither one of them. You can directly link pictures and videos via the href attribute. Note that since Next.js 9.1 the standard folder for static files is called **public** instead of **static**. You can reach to files inside that folder via root. 

Here's an example for a video:

```
<video className="video hidden md:block" poster="/video_prescreen.png" autoPlay playsInline muted loop>
  <source src="/video_bg.mp4" type="video/mp4" />
</video>
```

**Comment by jeremybarbet** [comment]
Should be fixed in `1.2.0` that now exposes a new prop to define the [assetDirectory](https://github.com/jeremybarbet/next-videos#assetdirectory)
